### ISSUE TITLE
Issue with Combine on iOS 12

### ISSUE THREAD

**Issue by fredpi** [Moya/Moya#1914]
In an app of mine, I included Moya Version `14.0.0-beta2` via Accio (As Accio is built on Carthage, I effectively installed it via Carthage).

Running my app on iOS 13 (Simulator) works fine, but when I run on my iPhone with iOS 12, the app crashes on launch and Xcode logs the following error message:

```
dyld: warning: could not load inserted library '__PLATFORMS__/iPhoneOS.platform/Developer/usr/lib/libXCTestBundleInject.dylib' into hardened process because image not found
dyld: Library not loaded: /System/Library/Frameworks/Combine.framework/Combine
  Referenced from: /private/var/containers/Bundle/Application/10FE3E5A-2604-473C-ACB1-B52AD8B6E469/Debug.app/Frameworks/Moya.framework/Moya
  Reason: image not found
```

Seemingly, Combine is expected, but, of course, it's not available when running iOS 12.

As Moya is already pre-compiled and doesn't know which iOS version it will be deployed to at compile-time, the Combine framework must somehow be made "optional" and not required in order to keep iOS 12 compatibility...

**Comment by KevinLaRosa** [comment]
Same on carthage with the `14.0.0-beta.1` or  `14.0.0-beta.2`. 
Right now, it seems impossible to compile Moya for iOS 12 on the command line tools Xcode 11.0 with carthage because of the Combine Framework link.

**Comment by sunshinejr** [comment]
@KevinLaRosa yeah it's not an easy task to solve. But we will prepare a fix for Xcode 11 users that should resolve it (unfortunately making Xcode 10 + Carthage + Moya 14.* users with one more step to correctly install Moya until we figure out a better way).

**Comment by KevinLaRosa** [comment]
Thank you for your answer, I'm waiting for your fix.
I will try to manage my project with both Xcode 10 and 11 😅 for a couple of days.

**Comment by floskel** [comment]
Do you know when the PR is planned on being released?

**Comment by sunshinejr** [comment]
@floskel this was just released with Moya 14.0.0-beta.3. Please try this one and let us know if you still have problems. Please remember that using Xcode 10 you will need [extra steps](https://github.com/Moya/Moya/tree/development#moya-14-carthage-and-xcode-10) to get this one to work. Also don't hesitate to reopen this issue if you the issue persists.
